.github/workflows/node-ci-pr.yml

```diff
@@ -104,22 +89,18 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-latest]
```

Comment:
This did not have the node-version and so was creating a cache key starting with just Linux-